packages/text-input/package.json

```diff
@@ -0,0 +1,22 @@
+
+  {
+    "name": "@leafygreen-ui/text-input",
+    "version": "0.9.0",
+    "description": "leafyGreen UI Kit Text Input",
+    "browser": "./dist/index.web.js",
+    "main": "./dist/index.node.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+      "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js"
+    },
+    "license": "Apache-2.0",
+    "publishConfig": {
+      "access": "public"
+    },
+    "devDependencies": {
```

Comment:
Check to make sure that you're using the correct/latest version of each component. Some of these components aren't at version 4.0.0 yet which is causing your build to fail with our CI tool